src/main/java/org/terasology/climateConditions/HyperthermiaLevelChangedEvent.java

```diff
@@ -0,0 +1,10 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.climateConditions;
+
+public class HyperthermiaLevelChangedEvent extends ThermiaLevelChangedEvent{
```

Comment:
Please add JavaDoc directly from the start. When is this event supposed to be sent, what does it indicate, etc.?